pkgs/by-name/ap/apache-airflow3/package.nix

```
@@ -0,0 +1,246 @@
+{
+  fetchFromGitHub,
+  fetchPypi,
+  python3,
+}:
+
+let
+  python = python3.override {
+    self = python;
+    packageOverrides = pySelf: pySuper: {
+      connexion = pySuper.connexion.overridePythonAttrs (o: rec {
+        version = "2.14.2";
+        src = fetchFromGitHub {
+          owner = "spec-first";
+          repo = "connexion";
+          rev = "refs/tags/${version}";
```

You're already using `tag` mostly consistently elsewhere, so let's do here as well:


```suggestion
          tag = version;
```